### PR TITLE
finish 106

### PR DESCRIPTION
**Pull request by JoannaCode**
@FreeTymeKiyan

### CODE REVIEW

File: Medium/ConstructBinaryTree.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        }
+        TreeNode root = new TreeNode(postorder[postR]);
+        int loc = map.get(postorder[postR]);
+        root.left = helper(inorder, inL, loc - 1, postorder, postL, postL + loc - inL - 1, map);
+        root.right = helper(inorder, loc + 1, inR, postorder, postR - inR + loc, postR - 1, map);
+        return root;
+    }
+
+    public static void main(String[] args) {
+        // TODO Auto-generated method stub
```
Comment (FreeTymeKiyan) [inline Medium/ConstructBinaryTree.java]: please remove unnecessary comments

File: Medium/ConstructBinaryTree.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        }
+        return helper(inorder, 0, inorder.length - 1, postorder, 0, postorder.length - 1, map);
+    }
+    
+    public static TreeNode helper(int[] inorder, int inL, int inR, int[] postorder, int postL, int postR, HashMap<Integer, Integer> map){
+        if(inL > inR || postL > postR){
+            return null;
+        }
+        TreeNode root = new TreeNode(postorder[postR]);
+        int loc = map.get(postorder[postR]);
```
Comment (FreeTymeKiyan) [inline Medium/ConstructBinaryTree.java]: nice map usage, otherwise would need to traverse the array

**Comment by JoannaCode** [comment]
@FreeTymeKiyan  fixed~

**Comment by FreeTymeKiyan** [comment]
LGTM

**Pull request merged**